sdk/communication/Azure.Communication.ProgrammableConnectivity/README.md

```diff
@@ -0,0 +1,49 @@
+# Azure ProgrammableConnectivity client library for .NET
+
+This SDK allows .NET applications to interact with Azure Programmable Connectivity Gateway, instead of using `curl` requests. For information on APC, see [docs](https://learn.microsoft.com/en-us/azure/programmable-connectivity/).
+
+  [Source code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/src) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Communication.ProgrammableConnectivity) | [API reference documentation](https://azure.github.io/azure-sdk-for-net) | [Product documentation](https://docs.microsoft.com/azure)
+
+## Getting started
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/):
+
+```dotnetcli
+dotnet add package Azure.Communication.ProgrammableConnectivity --prerelease
+```
+
+### Prerequisites
+
+Before you can use the SDK successfully, you'll need to
+* Follow the [guide](https://learn.microsoft.com/en-us/azure/programmable-connectivity/azure-programmable-connectivity-create-gateway) to create a gateway, or have one already.
+* Note down your endpoint and `apc-gateway-id`, which is retrieved by following the guide linked.
+
+### Authenticate the client
+
+The SDK uses `DefaultAzureCredential` to authenticate with APC. See [docs](https://learn.microsoft.com/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet) for information on how to allow this to access your credentials.
+
+## Examples
+
+You can familiarize yourself with different APIs using [Samples](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples).
+
+### Key concepts for each example
+
+For each call that you make to APC with the SDK, you will follow the same pattern:
+* Create a client `baseClient = new ProgrammableConnectivityClient()`
+* Access the sub-client for your use case (sim-swap/location/number-verification/device-network) by calling say `baseClient.GetSimSwapClient()`
+* Create the content for your request by using the objects given by the SDK, for example `SimSwapVerificationContent`
+* Call the client with the content you've created
+* Access the result
+
+When handling an error, catch `RequestFailedException`, and log out the details. See [README.md](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples/README.md) for a specific example.
```

Comment:
Your top 2-3 champion scenarios for "Hello World" use need to be represented on the README in full.   The goal is to allow developers to find success without needing to leave the overview page.